R/db2.R

```diff
@@ -442,6 +442,29 @@ report_data_import <- function(con, name, id, config) {
                       append = TRUE)
   }
 
+  ## DB instance:
+  if (!is.null(dat_rds$meta$instance)) {
+    ## Only add row where instance is set
+    build_row <- function(type) {
```

Comment:
I think there's a little more coersion here than is safe, and in particular I think this might not work as expected on 3.6.x due to treatment of factors. Consider:

```
if (is.null(instance)) {
```

Reply:
(with data_frame being our internal wrapper that does `data.frame(..., stringsAsFactors = FALSE)`

Reply:
That's way nicer, thanks